### ISSUE TITLE
Questions about the DatagridView component

### ISSUE THREAD

**Issue by hbadi** [ComponentFactory/Krypton#245]
Hi,

 I'm quite frustrated about datagrid : 
- I'd like to use the datagrid component but with a one hiearchical column and other columns "flat". Is there a way to do that ?
- How can I constrain a numerical value to be either an integer or a bounded number (max/min) ?
- How can I group by a column or filter using classical boolean conditions ?
- Is it possible to dynamically hide/unhide a column through a settings dialog ?

Thanks.

**Comment by Wagnerp** [comment]
Hi @hbadi 

What version of the toolkit are you using?

**Comment by lesandrog** [comment]
Hello, enjoying the post, DatagridView does not have the ".BorderStyle" option. Example .BorderStyle = BorderStyle.Fixed3D

**Comment by Smurf-IV** [comment]
This is the "Old" factory...
So you might be able to wrap the Grid in a Panel and use the border style on that,
Or, Use Styled Borders elements directly (Not part of the DataGrid)

**Comment by blackholeearth** [comment]
> This is the "Old" factory...

İs there a new version?
İs this repo outdated??

**Comment by PWagner1** [comment]
> @Smurf-IV
> 
> > This is the "Old" factory...
> 
> İs there a new version? İs this repo outdated??
> 
> **Edit:** i think, you are referring to this, right? https://github.com/Krypton-Suite/Standard-Toolkit
> 
> İts also coming on top, at the nuget search. Got confused for a second.. 👌🤣 🤦

@blackholeearth 

Yes,  the CF version is no longer being developed, but our version is.
